### PR TITLE
Add playing with time discussion to python

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-core/playing-with-time/playing-with-time-discussion.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/python/python-core/playing-with-time/playing-with-time-discussion.md?ref=3009220bdce114cbddfe722b4345f0b14c4bf12c)

File: python/python-core/playing-with-time/playing-with-time-discussion.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+# Test Yourself
+
+---
+
+## Content
+
+Now that you've learned about time and date modules, it's time to practice!
+
+How would you create a program where you take a date using a module and separately print out all the different formats it provides?
```
Comment (nem035) [inline python/python-core/playing-with-time/playing-with-time-discussion.md:21]: Should it be "take" a date or maybe "create" a date, using the "datetime" module?

```suggestion
How would you create a program where you create a date using the `datetime` module and separately print out all the different formats it provides?
```

To me, "take" primarily implies creating a function that takes arguments, not necessarily taking a date using a module.

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**